llvm/test/CodeGen/EVM/fallthrough.mir

```diff
@@ -0,0 +1,45 @@
+# RUN: llc -x mir --start-after=evm-stackify < %s | FileCheck %s
+
+
+--- |
+
+  target datalayout = "E-p:256:256-i256:256:256-S256-a:256:256"
+  target triple = "evm"
+  define void @test_fallthrough() { ret void }
+
+...
+---
+# CHECK: PUSH4 @.BB0_1
+# CHECK-NEXT: JUMPI
+# CHECK-NEXT: PUSH4 @.BB0_2
+# CHECK-NEXT: JUMP
+# CHECK-NEXT: .BB0_1:
+
+name: test_fallthrough
+tracksRegLiveness: false
+machineFunctionInfo:
+  isStackified: true
+body:             |
+  bb.0:
+      JUMPDEST_S
+      PUSH_LABEL %bb.10 {
+        JUMPI_S
+      }
+      PUSH_LABEL %bb.13 {
+        JUMP_S
+      }
+
+  bb.10:
```

Comment:
Wow. Is it kinda EVM assembly?

Reply:
No quite. This is LLVM machine IR representation, but it's quite close to the assembler.